portal/config/eproms/QuestionnaireBank.json

```diff
@@ -262,7 +262,7 @@
     },
     {
       "classification": "recurring",
-      "due": "{\"days\": 0}",
+      "due": "{\"days\": 15}",
```

Comment:
Pretty fundamental change, it seems likely to have unintended consequences

Reply:
@pbugni Changes checked out on the UI, but just want to make sure, per Justin's comment, will the change here 
 (and elsewhere in the QuestionnaireBank.json) affect the global study?

Reply:
As discussed, this is very confusing as it is currently implemented.  The right thing to do is to address the internal state, as we use `due` and `start` as synonyms in some places and discrete values in others.

The change above alters the QuestionnaireBank definition for each respected case to have a different `due` date than before.  But those values are NOT used when looking at the `due` (better known as start or available) value for a timeline, assessment, etc.

I'm willing to pull together a refactor PR that addresses this, but it's a fair amount of work and change:
``` find portal -name \*.py| xargs grep due| grep -iv overdue| wc -l
121
```

Reply:
Per our convo this morning, I'm content w/ this change